disquaire_project/manager/models.py

```diff
@@ -20,11 +20,13 @@ def __str__(self):
         return self.name
 
 class Disk(models.Model):
+    # There is no unicity constraint and no index on the reference. Are you sure you need this?
```

Comment:
Unicity constraints go to the model. Then, in the views, on form submission, for validation will automatically raise an error when unicity constraints are violated.